qctrlopencontrols/dynamic_decoupling_sequences/predefined.py

```diff
@@ -369,33 +369,28 @@ def new_carr_purcell_sequence(
 
 
 def new_cpmg_sequence(
-    duration=None, number_of_offsets=None, pre_post_rotation=False, **kwargs
+    duration=1.0, number_of_offsets=1, pre_post_rotation=False, name=None
```

Comment:
Do you know if there are plans to change the parameter names in open-controls to our standard?

Reply:
Good question. I don't think there is one, but I think that is reasonable to do.
I would probably fix it after the refactoring of the DDS module is done.